src/lib/components/dashboard/workspace-grid/WorkspaceGrid.svelte

```diff
@@ -183,4 +164,8 @@
     font-size: 14px;
     color: white;
   }
+  .threedot-active {
+    visibility: visible;
```

Comment:
check if its taking space use d-none as class for hiding it from the view completely

Reply:
yes, but there is not such requirement to remove it from view